spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -89,24 +89,48 @@ List of available Stream Application Starters:
 [width="100%",frame="topbot",options="header"]
 |======================
 |Artifact Type |Stable Release |SNAPSHOT Release
-|RabbitMQ + Maven |link:http://bit.ly/Avogadro-SR1-stream-applications-rabbit-maven[http://bit.ly/Avogadro-SR1-stream-applications-rabbit-maven] |link:http://bit.ly/Bacon-BUILD-SNAPSHOT-stream-applications-rabbit-maven[http://bit.ly/Bacon-BUILD-SNAPSHOT-stream-applications-rabbit-maven]
-|RabbitMQ + Docker|link:http://bit.ly/Avogadro-SR1-stream-applications-rabbit-docker[http://bit.ly/Avogadro-SR1-stream-applications-rabbit-docker] | N/A ]
-|Kafka 0.9 + Maven 	  |link:http://bit.ly/Avogadro-SR1-stream-applications-kafka-09-maven[http://bit.ly/Avogadro-SR1-stream-applications-kafka-09-maven] |link:http://bit.ly/Bacon-BUILD-SNAPSHOT-stream-applications-kafka-09-maven[http://bit.ly/Bacon-BUILD-SNAPSHOT-stream-applications-kafka-09-maven]
-|Kafka 0.9 + Docker   |link:http://bit.ly/Avogadro-SR1-stream-applications-kafka-09-docker[http://bit.ly/Avogadro-SR1-stream-applications-kafka-09-docker] | N/A ]
-|Kafka 0.10 + Maven 	  |link:http://bit.ly/Avogadro-SR1-stream-applications-kafka-10-maven[http://bit.ly/Avogadro-SR1-stream-applications-kafka-10-maven] |link:http://bit.ly/Bacon-BUILD-SNAPSHOT-stream-applications-kafka-10-maven[http://bit.ly/Bacon-BUILD-SNAPSHOT-stream-applications-kafka-10-maven]
-|Kafka 0.10 + Docker   |http://bit.ly/Avogadro-SR1-stream-applications-kafka-10-docker[http://bit.ly/Avogadro-SR1-stream-applications-kafka-10-docker] | N/A ]
+
+|RabbitMQ + Maven
+|http://bit.ly/Avogadro-SR1-stream-applications-rabbit-maven
+|http://bit.ly/Bacon-BUILD-SNAPSHOT-stream-applications-rabbit-maven
+
+|RabbitMQ + Docker
+|http://bit.ly/Avogadro-SR1-stream-applications-rabbit-docker
+|N/A
```

Comment:
Ah, in Docker land it is either latest or versioned tags. There's no concept of SNAPSHOTs if I'm not wrong. cc:/ @trisberg